### DISCUSSION TITLE
Default Transit Option Selection

### DISCUSSION THREAD

**Discussion by msafdaar** [organicmaps/organicmaps#9221] [Ideas]
Currently, when selecting the start and end points of a route, OrganicMaps defaults to searching for a metro route first, even in areas where metro or train services are unavailable. This often results in a failed search, requiring me to manually choose my preferred mode of transit (car, walking, etc.). I suggest adding an option in the settings to allow users to select their default or preferred mode of transit. This would save time and enhance the user experience, especially for those in remote areas with limited transit options.

**Comment by biodranik** [comment]
You've enabled the subway layer but forgot to turn it off. Please press the top left button, and disable the subways layer.
